tools/accuracy_checker/dataset_definitions.yml

```diff
@@ -385,6 +385,13 @@ datasets:
       data_dir: Market-1501-v15.09.15
     annotation: market1501_reid.pickle
 
+  - name: globalme-reid
+    data_source: GlobalMe-reID
```

Comment:
according dataset location, data_source should be globalme-reid/GlobalMe-reID
and data_dir for converter also.

Reply:
Fixed.